pylops/basicoperators/FunctionOperator.py

```diff
@@ -65,8 +69,7 @@ def __init__(self, f, *args, **kwargs):
             self.dtype = kwargs["dtype"]
         except KeyError:
             self.dtype = "float64"
-        self.explicit = False
-
+        super().__init__(explicit=False, clinear=True, name=kwargs.get("name", None))
```

Comment:
Why not provide "\mathrm{Fun}" or "F" as default?

Reply:
"F" is good :)